_health-care/_js/utils/validations.js

```diff
@@ -12,6 +12,14 @@ function isValidDate(day, month, year) {
     date.getFullYear() === Number(year);
 }
 
+function isValidName(value) {
+  return value === '' || /^[a-zA-Z '\-]+$/.test(value);
+}
+
+function isValidRequired(value) {
```

Comment:
Gotcha.  The name is a bit confusing.  You seem to just be checking that the field is not empty.

Should this predicate function be named IsNotEmpty or hasValue or something?

In general, it's better to name the function after what it does as opposed to where it's used...